### PR TITLE
Add native language names

### PR DESCRIPTION
**Pull request by petk**
Hello, this patch includes the native language names for all translations in the `README.md` and `welcome.md` files. This is probably more logical from the native speaker's point of view. Sorted alphabetically and with English being on top.

Thank you for considering merging it. In case you want some adjustments for your translations, let me know.

### CODE REVIEW

**Comment by meye** [comment]
@petk Thanks! I think it seems to be done while translating sometime. But, I would like to merge your patch if you add one more change in your patch - removing line 36, which describes the list order for Korean translation. It's not necessary anymore as the patch applied.

**Comment by petk** [comment]
Sounds logical. Self-referenced translation has been removed in above commit. If there is anything more to add or change, just let me know. Thanks.

File: README.md [hunk 1]
```diff
@@ -35,27 +35,28 @@ _PHP: The Right Way_ 는 당신의 웹사이트에 사용할 수 있는 웹 배
 
 (역주 : 원래 영문순이라 한글 순으로 다시 정렬하였습니다. 이후 추가된 경우 순서가 다를 수 있으니 주의하시기 바랍니다.)
```
Comment (meye) [inline README.md]: This line should be removed for this patch as well.

**Comment by meye** [comment]
I think there was some confusion. I meant that description text at line 36 should be removed, not Korean translation link. I added a comment on your original patch to make it clear.

**Comment by petk** [comment]
@meye ah, that. Well, I don't speak Korean so kind of difficult to tell... :D

**Pull request merged**